### DISCUSSION TITLE
SSL & TLS Issues

### DISCUSSION THREAD

**Discussion by onedogsolutions** [postalserver/postal#1979] [Help with using Postal]
I've tried to amend my config files as in some of the other discussions with the RSA key update to 4096, and symlinking the certs into the /opt/postal/config directory, but haven't been able to access the postal dashboard securely or send messages with TLS. Fresh install this week following the instructions here: https://docs.postalserver.io

**Comment by uhlhosting** [comment]
@onedogsolutions 
Can you output the logs from Web / Worker / SMTP ? Please check previous discussion on how to check logs.

**Reply by onedogsolutions** [reply]
Start of the web log:

postal-web-1  | Puma starting in single mode...
postal-web-1  | * Puma version: 5.6.2 (ruby 2.6.9-p207) ("Birdie's Version")
postal-web-1  | *  Min threads: 5
postal-web-1  | *  Max threads: 5
postal-web-1  | *  Environment: production
postal-web-1  | *          PID: 9
postal-web-1  | * Listening on http://152.44.39.97:5000

**Reply by onedogsolutions** [reply]
In the config, it's specified to use https:

  GNU nano 4.8                                                                                                postal.yml                                                                                                          
general:
  This can be changed to allow messages to be sent from multiple IP addresses
  use_ip_pools: false

web:
   The host that the management interface will be available on
  host: postal.onedog.solutions
   The protocol that requests to the management interface should happen on
  protocol: https

**Reply by onedogsolutions** [reply]
web_server:
 Specify configuration for the Postal web server
  bind_address: 152.44.39.97
  port: 5000

**Comment by willpower232** [comment]
> symlinking the certs into the /opt/postal/config directory

this probably won't work as the container filesystem won't be able to follow the link to the eventual file, you'll have to copy the certificate file in.

**Reply by onedogsolutions** [reply]
I copied the certificate files that Caddy generated and specified them in the postal.yml file, but still could not use TLS.

**Reply by willpower232** [reply]
https://github.com/postalserver/postal/discussions/1851#discussioncomment-2713544
